zmon_aws_agent/postgresql.py

```diff
@@ -58,3 +59,124 @@ def get_databases_from_clusters(pgclusters, infrastructure_account, region,
         logger.exception("Failed to make Database entities for PostgreSQL clusters!")
 
     return entities
+
+
+def collect_addresses(infrastructure_account):
+    ec2 = boto3.client('ec2')
+
+    addresses = call_and_retry(ec2.describe_addresses)['Addresses']
+
+    return [a for a in addresses if a['NetworkInterfaceOwnerId'] == infrastructure_account.split(':')[1]]
+# FIXME: depend on region, too?
+
+
+def collect_asgs(infrastructure_account):
+    asg = boto3.client('autoscaling')
```

Comment:
can't we hook on `aws.py:get_auto_scaling_groups()?` instead of calling describe_auto_scaling_groups twice?